cmd/namespace.go

```diff
@@ -1,26 +1,30 @@
 package cmd
 
 import (
+	"errors"
+
 	"github.com/odpf/optimus/config"
 
 	"github.com/AlecAivazis/survey/v2"
 	"github.com/odpf/salt/log"
 )
 
-func askToSelectNamespace(l log.Logger, conf config.Optimus) *config.Namespace {
+func askToSelectNamespace(l log.Logger, conf config.Optimus) (*config.Namespace, error) {
 	options := make([]string, len(conf.Namespaces))
 	for i, namespace := range conf.Namespaces {
 		options[i] = namespace.Name
 	}
+	if len(options) == 0 {
+		return nil, errors.New("no namespace found in config file")
+	}
```

Comment:
Can you please check the length of `conf.Namespaces` and return before the loop if the slice is empty.

Reply:
sure, done